documentation/sdkautomation/GenerateOutputSchema.json

```diff
@@ -46,6 +46,19 @@
             "type": "string"
           }
         },
+        "typespecProject": {
+          // List of related typespec project of this package.
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "language": {
+          "type": "string"
+        },        
+        "apiViewArtifact": {
```

Comment:
Modular client may contains multiple api views([example](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/mongocluster/arm-mongocluster/review)). Could you change it to string array?

Reply:
Good point. @qiaozha , can you confirm if this case will create multiple API views? I don't find a good example in the pipeline run with multiple API view artifacts generated.

Reply:
> I don't find a good example in the pipeline run with multiple API view artifacts generated.
Pipeline is not supported modular client generation for now, so you can't find an example in pipeline. the generated mongocluster sample I shared is ran by vendor manually.

Reply:
@wanlwanl 
Thanks for the clarification. This may need the support from API view creation too. We will create the feature request accordingly.